invoicer_project/static/js/index.js

```diff
@@ -0,0 +1,36 @@
+// Menu
```

Comment:
here you are using jQuery?

Reply:
The menu code was taken from the Codepen and changed to our needs
(Yes, there is used jQuery)